cool-demos/slash-commands.html

```diff
@@ -0,0 +1,247 @@
+<!--
+    This demo showcases how to use the autocompleter plugin to make a
+    slash commands feature and wrap it in a custom plugin
+
+    Learn more about making custom plugins at https://www.tiny.cloud/docs/advanced/creating-a-plugin/
+
+    Please note this demo contains premium features
+-->
+
+<!doctype html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Slash commands plugin</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="https://cdn.tiny.cloud/1/qagffr3pkuv17a8on1afax661irst1hbr4e6tbv888sz91jc/tinymce/5-dev/tinymce.min.js" referrerpolicy="origin"></script>
+
+    <script>
+        // Register a TinyMCE plugin
+        // https://www.tiny.cloud/docs/advanced/creating-a-plugin/#registeringacustompluginwithtinymce
+        tinymce.PluginManager.add('commandpalette', function (editor) {
+
+            // Instead of defining a custom icon pack, we can register individual
+            // icons using the API.
+            // https://www.tiny.cloud/docs/api/tinymce.editor.ui/tinymce.editor.ui.registry/#addicon
+            editor.ui.registry.addIcon('mention', '<svg width="21" height="20" xmlns="http://www.w3.org/2000/svg"><g><path d="M10.083 9.126a.207.207 0 0 0 .073-.195.21.21 0 0 0-.133-.161l-.442-.162-.523-.191a.488.488 0 0 1-.115-.258 2.19 2.19 0 0 1 .049-1.083 4.682 4.682 0 0 0 1.221-3.595C10.213 1.463 8.898 0 7.083 0c-1.814 0-3.13 1.463-3.13 3.478a4.667 4.667 0 0 0 1.214 3.58c.117.355.138.734.06 1.099a.493.493 0 0 1-.113.255l-.526.195c-2.039.75-3.511 1.29-3.961 2.189A7.633 7.633 0 0 0 0 13.75c0 .23.187.417.417.417h6.91a.208.208 0 0 0 .208-.19 7.063 7.063 0 0 1 2.548-4.851zm4.437-.376a5.632 5.632 0 0 0-5.622 6.928 5.56 5.56 0 0 0 4.155 4.167 5.72 5.72 0 0 0 2.769-.037.833.833 0 0 0 .591-1.019.823.823 0 0 0-.866-.6.795.795 0 0 0-.154.01 4.193 4.193 0 0 1-1.967.022 3.937 3.937 0 0 1-2.903-2.913 4 4 0 0 1 .789-3.444 3.934 3.934 0 0 1 3.166-1.449 4.078 4.078 0 0 1 3.834 3.934c0 .083.022.157.022.238v.683a.667.667 0 0 1-1.328 0v-.896a2.63 2.63 0 1 0-2.63 2.63c.43 0 .852-.109 1.23-.314a.207.207 0 0 1 .26.052c.282.34.654.595 1.075.733.234.082.48.126.729.128a2.333 2.333 0 0 0 2.333-2.333v-.676A5.777 5.777 0 0 0 14.52 8.75zm-.145 6.583a.964.964 0 1 1 .963-.964.964.964 0 0 1-.963.97v-.006z"/></g></svg>')
```

Comment:
Ah, correct, I removed the mention feature because it didn't have any people to mention and the demo became overloaded if I would have included all that too. I'll remove it.